ob-watcher.py

```diff
@@ -1,5 +1,4 @@
 import BaseHTTPServer, SimpleHTTPServer, threading
-from decimal import Decimal
```

Comment:
In common.py we have ```from decimal import Decimal```, and a few lines down we have ```from common import *```, making the above import redundant.